_1_basics/src/main/java/code/_4_student_effort/FizzBuzz.java

```diff
@@ -0,0 +1,34 @@
+package code._4_student_effort;
+
+public class FizzBuzz {
+    /* prints the numbers from 1 to 100
+       but for multiples of
+            * 3 prints "Fizz"
+            * 5 prints "Buzz"
+            * both (3 & 5) prints "FizzBuzz"
+            * 7 prints "Rizz"
+            * 11 prints "Jazz"
+     */
+
+    public static void main(String[] args) {
+        for (int i = 1; i <= 100; i++) {

```

Comment:
pentru numarul 105, care este divizibil si cu 3,5,7 ar trebui sa afiseze FizzBuzzRizz

cum am proceda in cazul asta a.i. sa reducem numarul de conditii adaugate ?

Reply:
Cred ca o implementare mai generala m-ar ajuta, am vazut apoi si codul deja implementat si cred ca folosirea claselor si modularizarea lui ar ajuta mult la mentenabilitatea codului.